purchase_last_price_info/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 
 {
     "name": "Purchase Product Last Price Info",
-    "version": "12.0.1.0.0",
+    "version": "12.0.1.1.0",
```

Comment:
Remove this changes. It will taken care by merge command.

Reply:
> Remove this changes. It will taken care by merge command.

Done!